addon/styles/addon.scss

```diff
@@ -12,148 +12,183 @@ $form-bkgrnd: rgba(0, 0, 0, 0);
+    animation-name: remodal-to-up;
+  }
+}
+
+.remodal-wrapper {
   display: flex;
-  flex: 1;
-  flex-direction: column;
-  max-height: calc(100vh - #{$modal-padding});
-  max-width: 550px;
-  overflow: hidden;
-  padding: 0;
-  position: relative;
-  width: 550px;
-
-  .input-header {
-    border-bottom: $info-bar-border-width solid $frost-color-lgrey-1;
-    height: $info-bar-height + $info-bar-border-width;
-    padding: $info-bar-padding-top-bottom $info-bar-padding-left-right;
-    position: absolute;
-    top: 0;
-    width: 100%;
-    z-index: 999;
-
-    &.header-scrolled {
-      border-bottom: 0;
-      box-shadow: 0 2px 3px $scroll-box-shadow;
-
-      &:after {
-        background: $frost-color-white;
-        bottom: 0;
-        content: '';
-        height: $info-bar-height + $info-bar-border-width;
-        left: 0;
-        opacity: .95;
-        position: absolute;
-        right: 0;
-        top: 0;
-        z-index: -1;
+  align-items: center;
+  justify-content: center;
+
+  .ember-remodal {
+    &.window {
+      position: relative;
+      width: 550px;
+      max-width: 550px;
+      max-height: calc(100vh - #{$modal-padding});
+      padding: 0;
+      border: 0;
+      background-color: $frost-color-white;
+      box-shadow: 0 0 20 $modal-box-shadow;
+      overflow: hidden;
+
+      .ember-remodal {
+        &.yielded {
+          &.content {
+            display: flex;
+            flex-direction: column;
+
+            .input-header {
+              display: flex;
+              position: absolute;
+              top: 0;
+              flex-direction: column;
+              align-items: flex-start;
+              width: 100%;
+              height: $info-bar-height + $info-bar-border-width;
+              padding: $info-bar-padding-top-bottom $info-bar-padding-left-right;
+              border-bottom: $info-bar-border-width solid $frost-color-lgrey-1;
+              z-index: 999;
+
+              &.header-scrolled {
+                border-bottom: 0;
+                box-shadow: 0 2px 3px $scroll-box-shadow;
+
+                &:after {
+                  position: absolute;
+                  top: 0;
+                  right: 0;
+                  bottom: 0;
+                  left: 0;
+                  height: $info-bar-height + $info-bar-border-width;
+                  background: $frost-color-white;
+                  content: '';
+                  opacity: .95;
+                  z-index: -1;
+                }
+              }
+
+              .primary-title {
+                padding-bottom: 3px;
+                color: $frost-color-night-2;
+                font-size: $frost-font-m;
+              }
+
+              .sub-title {
+                color: $frost-color-grey-5;
+                font-size: $frost-font-s;
+              }
+            }
+
+            .frost-bunsen-container {
+              margin-bottom: $action-bar-height + $form-margin-bottom;
+            }
+
+            .frost-bunsen-form {
```

Comment:
I'd prefer to keep the interface on `modal-dialog` really simple since there shouldn't be a lot of customization allowed.  The intention of that addon is to show short messages in 1 of 4 formats